### PR TITLE
#24 remove AuthenticationHandler from core

### PR DESCRIPTION
**Pull request by fkroockmann**
AuthenticationHandler is too specialized to be in core.

### CODE REVIEW

File: src/tb/component/authenticationhandler/main.js [hunk 1]
```diff
@@ -16,10 +16,10 @@
  * You should have received a copy of the GNU General Public License
  * along with BackBuilder5. If not, see <http://www.gnu.org/licenses/>.
  */
+
 define(
-    'tb.core.AuthenticationHandler',
+    'tb.component/authenticationhandler/main',
```
Comment (ndufreche) [inline src/tb/component/authenticationhandler/main.js]: It's too specialized to be a part of component.
The right position it's in main application.

File: src/tb/init.js [hunk 2]
```diff
@@ -65,8 +65,8 @@ define(['jquery'], function (jQuery) {
                          */
                         Core.initConfig(config);
                     });
-
-                    Core.authentication.showForm();
+                    console.log(AuthenticationHandler);
```
Comment (ndufreche) [inline src/tb/init.js]: forgotten console.log

File: src/tb/component/authenticationhandler/main.js (hunk 1 above)
Comment (fkroockmann) [inline src/tb/component/authenticationhandler/main.js]: Main application is loaded when a user is connected not before.

Comment (ndufreche) [inline src/tb/component/authenticationhandler/main.js]: Ok, then this is a miss conception from the core.

**Comment by ndufreche** [comment]
Thanks for your job. Please squash your commits
:+1:

**Comment by fkroockmann** [comment]
Thanks for review @ndufreche, squash is done.

**Comment by mickaelandrieu** [comment]
:+1:

**Pull request merged**